pom.xml

```diff
@@ -678,5 +677,14 @@
         <groups>slow,flaky</groups>
       </properties>
     </profile>
+    <profile>
+      <id>camel</id>
+      <activation>
+        <jdk>17</jdk>
```

Comment:
I'm wondering if we need to configure the java target / source. 

If it stays like that, we need to see how we can release stuff without making it too complicated. Maybe all but camel and docs should use 11 as source and target, and 17 for the rest. We keep the CI build as it is currently (with your profile, but we may need to do the same for docs).

Then we cut releases using java 17 (I think we do already because of revapi).

Reply:
That makes sense, let's start releasing on java 17.